target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -16,7 +16,9 @@ do_sysinfo_octeon() {
 	"UBNT_E220"*)
 		name="erpro"
 		;;
-
```

Comment:
Just don't touch those empty lines, keep the idea of the initial formatting (or remove the empty lines between the options above as well).